committees/project-structure.md

```diff
@@ -0,0 +1,55 @@
+# Rust Project Structure Committee
+
+## Mission
+
+The *Rust Project Structure* committee is tasked with determining the "shape" of the Rust Project and its membership,
+and to make proposals to the Council on any recommended changes,
+and to produce documentation on clarifying the Project structure.
+
+## Goals
+
+The goals of the committee are to address the following concerns, and to determine if there are any changes that should be proposed to the Council:
+
+- Review the structure of the Project. <https://github.com/rust-lang/leadership-council/issues/33>
+- Propose guidelines for determining whether or not to accept a new team in the project.
+  In essence, more clearly answer the question of "what is the Rust Project?".
+- Make recommendations on organizational cleanup.
+    - Identify concerns around the current structural schemes.
+    - Ensure all teams and other governance structures are "attached" to appropriate places in the structure. This includes working with teams to find appropriate homes, and ensuring such changes are ultimately reflected in the team metadata repository.
+    - Re-organize the top-level teams. <https://github.com/rust-lang/leadership-council/issues/32>
+    - Find homes for teams in the launching-pad.
+- Collect and organize team charters. <https://github.com/rust-lang/leadership-council/issues/44>
+    - Every team or group should have a charter that defines certain properties about them (their purview, mission, membership, etc.).
+    - This is intended to feed back into the prior points, as it can be difficult in some cases to understand what the structure should be if we don't know what the purview or responsibilities of existing teams are.
+- Better define "team", "sub-team", "working-group", and "project group".
+    - https://rust-lang.github.io/rfcs/2856-project-groups.html
+    - https://github.com/rust-lang/wg-governance/blob/master/draft-rfcs/taxonomy.md
+    - And possibly determine if different structures or terminology is needed.
+- Propose a policy around inactive teams/groups.
+
+## Delegated Responsibilities
+
+No decision-making authority is given to this committee. It is only responsible for producing recommendations to the Council (and ultimately post RFCs to make changes), along with documentation of the existing structures.
+
+## Duration
+
+This is a temporary committee, intended to last as long as needed to accomplish the goals listed above. The committee may be retired earlier if they decide to reduce the scope of the goals.
```

Comment:
Would it be worth adding a review in six months or so to make sure the committee is either still working, or if it's finished make sure we wind it down?

Reply:
I think a review session is highly encouraged for groups like this in the Council RFC, but I agree we should be explicit here. This seems like a good habit to get into.